src/error.rs

```diff
@@ -3,34 +3,42 @@ use snafu::{Backtrace, Snafu};
 #[derive(Debug, Snafu)]
 #[snafu(visibility = "pub")]
 pub enum Error {
-    /// An error occurred while sending or receiving a HTTP request or response
-    /// respectively.
-    #[snafu(display("Source: {}\nBacktrace:\n{}", source, backtrace))]
-    Http {
-        source: reqwest::Error,
-        backtrace: Backtrace,
-    },
+	/// An error occurred while sending or receiving a HTTP request or response
```

Comment:
This PR is using a mix of tabs and spaces. We should probably just be using 4 spaces.

Reply:
I'm only just getting used to tabs, would appreciate sticking with them